share/getters/shrex_test.go

```diff
@@ -101,9 +101,11 @@ func TestShrexGetter(t *testing.T) {
 		})
 
 		// corrupt NID
-		nID[4]++
+		newNID := make([]byte, 8)
+		copy(newNID, nID)
+		newNID[4]++
```

Comment:
This one took me around 4 hours.

Reply:
omg, hate those days

Reply:
Why was it failing? It's not apparent why copy is needed